src/components/EditInfo.vue

```diff
@@ -21,6 +25,13 @@
                 clearable
                 label="Avatar" color="secondary"></v-text-field>
             </v-row>
+            <v-row>
+              <v-checkbox
```

Comment:
This should be a button. A checkbox should be able to revert the state to the last known state. In this case, it should be able to use the custom avatar field, but since we don't save the old value, it can't go back (re: it can't "toggle" the value.). If we really want a toggle here, we should use the gravatar in all places in the app, and check for it, instead of overwriting the avatar.